software/platformio.ini

```diff
@@ -1,32 +0,0 @@
-; PlatformIO Project Configuration File
```

Comment:
Please note that this file is FUCKING IMPORTANT!!!

Reply:
Oh dear your right bra